charts/sf-archival/charts/spark-manager/templates/configmap.yaml

```diff
@@ -142,9 +142,13 @@ data:
               "--name",
               "__PARAM__name",
               "--master",
-              "local[{{ .Values.infrastructure.EcsFargate.numExecutors }}]",
+              "local[{{ .Values.infrastructure.EcsFargate.sparkExecutorResources.numExecutors }}]",
```

Comment:
Did this line come up properly in config-map ?
I do not see that variable in values.yaml